Contact me form for my portfolio

D4nielJ opened the pull request:
## Hello Reviewer

During these commits, my coding partner and I, focus on the Contact Me form at the end of the portfolio. Only for mobile sizes for now.

Contact-me Form has the right validation: 

- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.

Contact-me form collects data using Formspree Service.

You can check the [Live Demo here.](https://d4nielj.github.io/developer-portfolio/)

Let me know any changes or suggestions.

Cheers,
Daniel Jaramillo

D4nielJ committed:
Add float style to tags

D4nielJ committed:
add contact-me form

D4nielJ committed:
add contact-me-form style

D4nielJ committed:
add background file

the-catalystmc committed:
Add styling and color to text and input boxes on contact form.

D4nielJ committed:
Add action link to contact form and final styling details to form
Link supported by Formspree service

D4nielJ committed:
Change Hover color in contact me button
Different shade so it doesn't get confused with the BG

Davidosky007 commented on index.html:
```diff
@@ -192,5 +191,33 @@ <h2>Skills</h2>
+                    <input type="email" class="form-input-box" name="user_email" placeholder="Enter your e-mail here" required>
+                </li>
+                <li>
+                    <textarea class="form-input-box form-input-box-area" placeholder="Write your message here" name="user_message" required maxlength="500"></textarea>
```
I suggest message input's type equals textarea as stated in the project requirement

D4nielJ replied:
`textarea` doesn't allow for `type="textarea"`, I think it's already implied by the tag. I tried doing it like you said. A Linter error came up.

Davidosky007 replied:
ok you can undo it am sorry its a mistake on our end 🙏 kindly fix the other changes and submit them for review 👍

Davidosky007 commented on index.html:
```diff
@@ -192,5 +191,33 @@ <h2>Skills</h2>
+        action="https://formspree.io/f/mjvjrbzg"
+        method="post"
+      >
+        <h1>Contact me</h1>
```
This is a subtitle i suggest you wrap it an `hx` tag and not `h1` tag. `h1` tag is used for main title get more information [Here](https://www.w3schools.com/tags/tag_hn.asp)

D4nielJ replied:
Done

Davidosky007 commented on css/styles.css:
```diff
@@ -334,6 +334,72 @@ header {
+  justify-content: center;
+  align-items: flex-start;
+  padding: 50px 5%;
+  background: #6070ff url(../media/contact-form-shapes-mobile.svg) 135% 50% no-repeat;
```
You can make the background-position same as the design template this is yours 
![Screenshot (25)](https://user-images.githubusercontent.com/52497246/121270105-f717c580-c8b8-11eb-86df-1e3676de6e57.png)

This is the design template please compare and make necessary adjustment 👍 
![Screenshot (24)](https://user-images.githubusercontent.com/52497246/121270188-20d0ec80-c8b9-11eb-82d9-aa3195c0228c.png)

D4nielJ replied:
Done

Davidosky007 reviewed (changes requested):
Hi @the-catalystmc @D4nielJ, 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest you start your commit message with capital letters
* I suggest your Pull Request should have a short summary describing introduced changes in more detail. please make it more detailed as stated in the project requirement 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

D4nielJ committed:
Fix Changes from Pull Request
Contact me tag H1 to H2
Add type="textarea" to message input
Background ajustments

D4nielJ committed:
Fix Linter error, type=textarea not allowed in textarea tag

mricanho reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @D4nielJ @the-catalystmc!

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

![shine team](https://user-images.githubusercontent.com/60631456/121384401-d9288000-c90d-11eb-95eb-ec3d3d744bcd.gif)

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
